cns/service/main.go

```diff
@@ -875,6 +875,12 @@ func main() {
 			if err = httpRemoteRestService.SavePnpIDMacaddressMapping(rootCtx); err != nil {
 				logger.Errorf("Failed to fetch PnpIDMacaddress mapping: %v", err)
 			}
+
+			// No-op for linux, setting primary macaddress if VF is enabled on the nics for aks swiftv2 windows
+			logger.Printf("Setting VF for accelnet nics if enabled")
```

Comment:
nit: even if this is no-op for Linux, this would still appear in the logs even though CNS is running in a Linux machine, which could be confusing for someone who doesn't have contexts. Maybe add some clarification?